torch/_functorch/aot_autograd.py

```diff
@@ -1874,12 +1876,20 @@ def backward(ctx, *flat_args):
 
             def call_compiled_backward():
                 if CompiledFunction.compiled_bw is None:
-                    # TODO - pass in fake tensors ?
-                    context = disable_autocast_manager if disable_amp else nullcontext
-                    with context(), track_graph_compiling(aot_config, "backward"):
-                        CompiledFunction.compiled_bw = aot_config.bw_compiler(
-                            bw_module, all_args
+                    if config.use_dynamic_shapes:
+                        all_args_list = list(all_args)
+                        CompiledFunction.compiled_bw = create_aot_dispatcher_function(
```

Comment:
What's the idea behind re-invoking `create_aot_dispatcher_function` now when dynamic shapes are enabled?